ansible.cfg

```diff
@@ -9,3 +9,6 @@ connection_plugins = plugins/connection
 callback_plugins = plugins/callbacks
 transport = monkeypatched_ssh
```

Comment:
Looks like this is no longer needed: https://github.com/blueboxgroup/ursula/blob/master/plugins/connection/monkeypatched_ssh.py, https://github.com/ansible/ansible/pull/5732/files